contrib/default.config

```diff
@@ -46,18 +46,27 @@ ST_LINUXBOOT_VARIANT=minimal
 # into the LinuxBoot initramfs.
 ST_SIGNING_ROOT="out/keys/example_keys/root.cert"
 
-# ST_HOST_CONFIG_LOCATION defines the location where the host configuration is
-# stored.
+# ST_HOST_CONFIG_MODE defines where the host configuration is stored.
 #
 # initramfs (default): Initramfs of LinuxBoot.
 #
 # cdrom (iso target only): ISO filesystem
 #
 # legacy (disk target only): STBOOT partition
 #
-# efivar: (U)EFI variable "STHostConfig-f401f2c1-b005-4be0-8cee-f2e5945bcbe7"
+# efivar: (U)EFI variable
+#
+ST_HOST_CONFIG_MODE="initramfs"
+
+# ST_HOST_CONFIG_LOCATION defines the location of the host configuration
+# depending on the config mode (ST_HOST_CONFIG_MODE). It's defines either the
+# path of the configuration file or in case it is set to "efivar" the EFI
+# variable name. The mode "legacy" does not support this configuration option.
 #
-ST_HOST_CONFIG_LOCATION="initramfs"
+# "initramfs" and "cdrom" mode:
+#ST_HOST_CONFIG_LOCATION="/etc/host_configuration.json"
+# "efivar" mode:
+#ST_HOST_CONFIG_LOCATION="STHostConfig-f401f2c1-b005-4be0-8cee-f2e5945bcbe7"
 
 # ST_HOST_CONFIG_FILE is the local path to the host configuration file.
```

Comment:
`ST_HOST_CONFIG_FILE is the local path where the host configuration file is taken from.

Reply:
ack